src/Utils/removeDuplicates.ts

```diff
@@ -0,0 +1,81 @@
+export const groupAndMergeByColumn = (
+  data: string,
+  groupByColumnIndex: number,
+  mergeColumnIndices: number[],
```

Comment:
there's no guarantee the column indice will be same everytime. if we decide to change something in backend (add/remove field), it'd be hard to realise this messed up.

instead use column name.

also i'd recommend passing named args instead of positional args.